### ISSUE TITLE
Games folder can get scrolled away and disappear with narrow windows

### ISSUE THREAD

**Issue by beholdnec** [magcius/noclip.website#274]
The button for the games folder can disappear in situations where the browser window is narrow. This commonly occurs on tablets in portrait mode.

Steps to reproduce:
1. Resize the window to be skinny horizontally
2. Open the games folder, then select a game and open a scene. Notice that the folder icon gets pushed left out of view.
3. Click the games header to close the games folder. The games icon is no longer visible and cannot be accessed anymore, even if the window is made wider.

**Comment by magcius** [comment]
Yeah, I've hit this before. There's a planned UI redesign in the works, so I'm not going to spend too much time improving the current UI.
